cmd/run.go

```diff
@@ -52,11 +57,36 @@ func newRunCMD(version string) *cobra.Command {
 				return err
 			}
 
-			client, err := resolver.PolicyReportClient()
+			discoveryClient, err := discovery.NewDiscoveryClientForConfig(k8sConfig)
 			if err != nil {
 				return err
 			}
 
+			var wgClient, orClient report.PolicyReportClient
+
+			_, err = discoveryClient.ServerResourcesForGroupVersion(orclient.OpenreportsReport.Group + "/" + orclient.OpenreportsReport.Version)
```

Comment:
this logic could be part of the resolver and only return a client when the resource is found